forge-gui/res/cardsfolder/upcoming/bandits_talent.txt

```diff
@@ -0,0 +1,14 @@
+Name:Bandit's Talent
+ManaCost:1 B
+Types:Enchantment Class
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDiscard | TriggerDescription$ When CARDNAME enters, enters, each opponent discards two cards unless they discard a nonland card.
+SVar:TrigDiscard:DB$ Discard | Defined$ Opponent | NumCards$ 2 | Mode$ TgtChoose | UnlessCost$ Discard<1/Card.nonLand/nonland card>

```

Comment:
I don't think that UnlessCost works since this isn't a "shared payment"

Reply:
Would it work like this?

Reply:
probably but this causes other problems since it's supposed to be simultaneous
UnlessType$ will be better for this case

Reply:
Darn, I don't like that one :)